doxia-core/src/test/java/org/apache/maven/doxia/sink/impl/AbstractSinkTest.java

```diff
@@ -644,10 +645,65 @@ public void testTable() {
         if (isXmlSink()) {
             assertThat(wrapXml(actual), isIdenticalTo(wrapXml(expected)));
         } else {
-            assertEquals(actual, expected);
+            assertEquals(expected, actual);
```

Comment:
Can you move these to a separate PR? Good catch, BTW.

Reply:
No, sorry, too much effort for just fixing a glitch in a test class.